### ISSUE TITLE
Secure tripcodes

### ISSUE THREAD

**Issue by ctrlcctrlv** [infinity-next/infinity-next#313]
This is a follow up to https://9chan.us/9/thread/24#457

When a tripcode with two hashes as ## is used, it's interpreted as a regular DES tripcode. This is likely to confuse users. Tripcodes beginning with a hash should be rejected as invalid until a secure tripcode is implemented.

As secure tripcodes are different on every site, there's no need to match the 4chan/8chan way of doing it (3DES + a salt). 

SHA256 could be used, or BLAKE2. Or cryptographic key generation and signing could be implemented in the browser, similar to this : https://www.youtube.com/watch?v=c8EjDKEeusM

**Comment by jaw-sh** [comment]
This is a duplicate of #28, you opened almost 5 years ago exactly, but I'll close that one now.

Since then, the original intention of the HTML5 (then in beta and barely supported) `keygen` tag has been deprecated in favor of a keygen attribute. I have never seen it used, ever.
https://www.w3schools.com/TAGS/att_keygen_name.asp

**Comment by ctrlcctrlv** [comment]
I love it. Never heard of it.

**Comment by milezzz** [comment]
Ohhh shit just noticed the pgp signing commit w/ gnupg_verify() for 0.6.0. Wow the need for this within the confines of an anonymous image board cannot be understated. So overdue. Absolutely capital Jawsh, well done!

**Comment by jaw-sh** [comment]
I'm waiting on a patch form PHP Security because the php-gnupg mod is just completely, totally shattered.
